README.md

```diff
@@ -18,7 +18,7 @@ Command line utility to quickly create a new adapter or VIS widget for [ioBroker
 
 ## Prerequisites
 
-Any computer with NodeJS in version 8 or higher and npm 6 or higher.
+Any computer with NodeJS in version 12 or higher and npm 6 or higher.
```

Comment:
Wer should not implicitly suggest that npm 7 is supported. I think, there should be an explicit mention that we currently *do not* support npm 7.

Reply:
Uhm why should the creator not be working on npm7?

Reply:
Agree, npm7 is currently only an issue when installing adapters or stuff from GitHub

Reply:
I've added a note.